src/hb-ot-var-hvar-table.hh

```diff
@@ -323,7 +326,7 @@ struct HVARVVAR
 				    const int *coords, unsigned int coord_count,
 				    VariationStore::cache_t *store_cache = nullptr) const
   {
-    uint32_t varidx = (this+advMap).map (glyph);
+    uint32_t varidx = advMap ? (this+advMap).map (glyph) : glyph;
```

Comment:
Why is this line needed? The null `DeltaSetIndexMap` is designed to handle that I think.

Reply:
Oh yes, I have reverted my change. Thanks!